test-apps/new-addon/tests/acceptance/snippets-test.js

```diff
@@ -0,0 +1,40 @@
+import { module, test } from 'qunit';
+import { visit, fillIn } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+const snippetIds = ['your-snippet-name.hbs', 'your-angle-bracket-snippet-name.hbs'];
+
+module('Acceptance | snippets', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('snippets support both classic & angle bracket invocation', async function(assert) {
+    assert.expect(13);
+
+    await visit('/snippets');
+
+    snippetIds.forEach(id => {
+      let snippet = `[data-test-id="${id}"]`;
+
+      assert.dom(`${snippet} pre`).includesText('<div id=\'foo\'>');
+      assert.dom(`${snippet} button`).hasAttribute('data-clipboard-text', /.*<div id='foo'>.*/);
+    });
+
+    let demo = '[data-test-id="your-angle-bracket-snippet-name.hbs"] ~ *:not([data-test-id])';
+    let secondDemo = `${demo}:last-child`;
```

Comment:
Let me note that I wanted to use the examples from the docs (or I'm just lazy) and the `<DocsDemo />` components don't provide with `data-test-id`s and I couldn't quickly find a better way to target them.